### ISSUE TITLE
Support folder recursion file checking.

### ISSUE THREAD

**Issue by rajinder-yadav** [palantir/tslint#2266]
### Bug Report

- __TSLint version__: 4.5.0
- __TypeScript version__: 2.2.1
- __Running TSLint via__: (pick one) CLI / Node.js API / VSCode / grunt-tslint / Atom / Visual Studio / etc

Not sure if this is broken, and should be added as a new feature. I would link tslint to pick up all files under a folder plus any sub-folder using this syntax:

```sh
tslint ./src/**/*.ts
```

This fails silently.

**Comment by adidahiya** [comment]
Globs are supported in the CLI using node-glob. Try quoting your glob pattern. See https://github.com/palantir/tslint/issues/692

**Comment by rajinder-yadav** [comment]
Thanks, can I suggest this be added to the README.
